src/styles/global.css

```
@@ -1 +1,20 @@
+  width: fit-content;
+  height: 50px;
+  padding: 15px 18px;
+  border-radius: 30px;
+  margin: auto auto 28px 16px;
+  background-color: #3195f3;
+  display: flex;
+  align-items: center;
+  position: relative;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+
+  color: #f9f6ef;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)